docs/08-developer-guide/04-project/03-release-guide.md

```diff
@@ -4,6 +4,372 @@ sidebar_label: Release Manager Guide
+
+## After-Vote
+
+### Publish the artifacts
+
+You should commit the artifacts to the SVN repository. If you are not a PMC member you can commit it to the dev zone first and ask a PMC for the final move.
+
+Checkout the svn folder &amp;commit the artifacts to a new directory.
+
+```
+svn checkout https://dist.apache.org/repos/dist/dev/ozone
+cd ozone
+svn mkdir "$VERSION"
+cp "$RELEASE_DIR"/* "$VERSION"/
+svn add "$VERSION"/*
+svn commit -m "Added ozone-$VERSION directory"
+```
+
+PMC members can move it to the final location:
+
+```
+svn mv -m "commit msg" https://dist.apache.org/repos/dist/dev/ozone/"$VERSION" https://dist.apache.org/repos/dist/release/ozone/"$VERSION"
+```
+
+To publish the artifacts to the central maven, login to [https://repository.apache.org/#stagingRepositories](https://repository.apache.org/#stagingRepositories)[,](https://repository.apache.org,) select your **staging** repository and **release** it.
+
+### Write a haiku
+
+Check the tag from the [Ozone Roadmap](https://cwiki.apache.org/confluence/display/OZONE/Ozone+Roadmap) page (it's a national park).
+Find a photo which is under the CC license.
+Write a haiku to the photo with Future font.
+
+### Update the ozone website
+
+ * Create release notes and add them to the Ozone website with your haiku image. An example pull request showing how to do this is [here](https://github.com/apache/ozone-site/pull/17). Note that the target branch is `master`.
+ * Extract the docs folder from the release tarball, and add its contents to the website. An example pull request for this is [here](https://github.com/apache/ozone-site/pull/18). Note that the target branch is `asf-site` , and that the docs/current symlink has been updated to point to the latest release's directory.
+ * Test the website locally by running `hugo serve` from the repository root with the master branch checked out. Check that links for the new release are working. Links to the documentation will not work until the PR to the `asf-site` branch is merged.
+
+### Add the final git tag and push it
+
+```bash title="Add final release tag"
+git checkout "ozone-$VERSION-RC$RC"
+git tag -s "ozone-$VERSION" -m "Ozone $VERSION release"
+git push origin "ozone-$VERSION"
+```
+
+### Update the master branch
+
+ * Cherry pick your commit updating the protolock files to a branch on your fork, and merge it to master with a pull request.
+ * Update the Ozone SNAPSHOT version and national park tag on master with a pull request. Here you will pick the [national park](https://en.wikipedia.org/wiki/List_of_national_parks_of_the_United_States) to use for the next release of Ozone and set it in the project's top level pom at &lt;ozone.release&gt;. [Example](https://github.com/apache/ozone/pull/2863)
```

Comment:
For the update of Ozone Snapshot version, the original recipe is to cut branch first, process the release, which might take about one month, then update the snapshot version. Then the commits between release-branch and snpashot version in that month seems becoming orphan.

And in release 1.4.0, I did update the snapshot version first, and then cut the branch based on the last commit of the snpashot version, which seems better for the orphan commits.